alfresco-googledrive-repo-base/src/main/java/org/alfresco/integrations/google/docs/webscripts/UploadContent.java

```diff
@@ -249,7 +248,7 @@ public void setVersionService(VersionService versionService)
 
             if (jsonStr == null || jsonStr.trim().length() == 0)
             {
-                throw new WebScriptException(HttpStatus.SC_BAD_REQUEST, "No content sent with request.");
+                throw new WebScriptException(SC_BAD_REQUEST, "No content sent with request.");
             }
             log.debug("Parsed JSON: " + jsonStr);
```

Comment:
Trivial: for consistency, wrap in an if block (as per rest of code) ?

Reply:
Done.